src/pages/StocktakeEditPage.js

```diff
@@ -1,564 +1,224 @@
-        } else {
-          cellContents = stocktakeItem.countedTotalQuantity;
-        }
-        // If finalised display 0 not '', as placeholder isn't shown.
-        return {
-          type: isEditable ? 'editable' : 'text',
-          cellContents,
-          placeholder: tableStrings.not_counted,
-        };
-      }
-      case 'difference': {
-        const { difference } = stocktakeItem;
-        const prefix = difference > 0 ? '+' : '';
-        return { cellContents: `${prefix}${difference}` };
-      }
-      case 'mostUsedReasonTitle': {
-        const canEdit = hasAnyReason && isEditable;
-        const onPress = this.openModal.bind(this, REASON_EDIT, stocktakeItem);
-        return (
-          <TouchableOpacity
-            key={stocktakeItem.id}
-            onPress={canEdit ? onPress : null}
-            style={localStyles.reasonCell}
-          >
-            {hasAnyReason && isEditable && (
-              <Icon name="external-link" size={14} color={SUSSOL_ORANGE} />
-            )}
-            <Text style={{ width: '80%' }} numberOfLines={1} ellipsizeMode="tail">
-              {stocktakeItem.mostUsedReasonTitle
-                ? stocktakeItem.mostUsedReasonTitle
-                : programStrings.not_applicable}
-            </Text>
-          </TouchableOpacity>
-        );
-      }
-      case 'modalControl': {
-        return (
-          <TouchableOpacity
-            onPress={() => {
-              this.setState({
-                isStocktakeEditModalOpen: true,
-                currentStocktakeItem: stocktakeItem,
-              });
-            }}
-          >
-            <View style={localStyles.modalControl}>
-              <Icon name="angle-double-up" size={20} color={SUSSOL_ORANGE} />
-            </View>
-          </TouchableOpacity>
-        );
-      }
+        return null;
     }
   };
 
-  renderManageStocktakeButton = () => {
-    const { navigateTo, stocktake } = this.props;
-
-    return (
-      <PageButton
-        text={buttonStrings.manage_stocktake}
-        onPress={() =>
-          navigateTo('stocktakeManager', navStrings.manage_stocktake, {
-            stocktake,
-          })
-        }
-        isDisabled={stocktake.isFinalised}
-      />
-    );
-  };
-
-  renderModalContent = () => {
-    const { stocktake, database } = this.props;
-    const { modalKey } = this.state;
-
-    const { NAME_EDIT, COMMENT_EDIT, REASON_EDIT } = MODAL_KEYS;
-
-    switch (modalKey) {
-      default:
-      case COMMENT_EDIT:
-        return (
-          <TextEditor
-            text={stocktake.comment}
-            onEndEditing={newComment => {
-              if (newComment !== stocktake.comment) {
-                database.write(() => {
-                  stocktake.comment = newComment;
-                  database.save('Stocktake', stocktake);
-                });
-              }
-              this.closeModal();
-            }}
+  const {
+    newPageTopSectionContainer,
```

Comment:
is there a plan to rename the 'new..' items when they aren't *new* anymore?

Reply:
yup #1165 - thought best to rename and remove later rather than change and break stuff - right move or too much extra work to be worth it?